docs/docs/index.md

```diff
@@ -29,6 +29,24 @@ curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poet
 ```powershell
 (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python -
 ```
+### dockerfile install instructions
+poetry.Dockerfile
+```Dockerfile
+FROM python:3.8-slim
+
+ENV POETRY_VERSION=1.1.3
+
+RUN apt-get update;apt-get install -y curl;rm -rf /var/lib/apt/lists/* \
+  ;curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python -
+```
+project.Dockerfile
+```Dockerfile
+FROM python:3.8-slim
+
+COPY --from=poetry:1 /root/.poetry /root/.poetry
+ENV PATH="/root/.poetry/bin:${PATH}"
+RUN poetry config virtualenvs.create false
```

Comment:
This is not something we recommend at all.

Reply:
I hate virtualenvs